api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -300,6 +300,18 @@ message ChannelSet {
 
   // The name used to identify the channel set (for example, TT&C UHF).
   string name = 2;
+
+  // The center frequency used for uplink, in Hz. Set to 0 if the channel set is downlink only.
+  //
+  // Status: ALPHA This API is under development and may not work correctly or be changed in backwards
```

Comment:
We can remove ALPHA since it will be officially supported.

Reply:
I marked these fields `ALPHA` because they are not guaranteed to be implemented at the time of merging. If someone were to use a snapshot, for example, these fields might not be set. Until they are guaranteed to be implemented I would like to keep the `ALPHA` status.

Similarly, it may be too premature to deprecate `downlink_center_frequency_hz` and `uplink_center_frequency_hz` since there is not yet a stable alternative.

Depending on when the next release of the API is we may be able to promote these fields to stable and deprecate the frequency fields.

Reply:
Got it. Let's change it then.